speechbrain/lobes/models/transformer/TransformerASR.py

```diff
@@ -152,9 +152,12 @@ def __init__(
             ),
             torch.nn.Dropout(dropout),
         )
-        self.custom_tgt_module = ModuleList(
```

Comment:
@asumagic I guess that this is the place of the problem?

Reply:
Yes, this change does what I ultimately suggested to do in the discussion in my PR, with the same consequences:

> Though, now, I'm thinking that we could just skip the headache and fix the problem by gating it behind `num_decoder_layers > 0` and break models: As far as I can tell, actually, the only affected model in the SB repo seems to be the conformer-transducer model.  
> Then, just in case, we could provide a script that would remove the module from a checkpoint file.